tsconfig.json

```diff
@@ -25,5 +25,8 @@
       "./node_modules/@types"
     ],
     "skipLibCheck": true
-  }
+  },
+  "exclude": [
```

Comment:
Re-add so it is part of the checks? (i.e. eslint that is a global project config)

Reply:
I had to validate the react-native project's typescript by treating exampleReactNative as its own package with it's own tsconfig.json, without workspaces. I think the `nohoist`ing interferes with the tsc --emitDeclarationOnly ... command in the polkadot-dev-build-ts script.

Reply:
Ahh, ok. Don't see this applying the same rulesets though. (since we skip at root and no explicit eslint file in here? And not called from the root `yarn lint` command either)

Reply:
I don't think this file is used for yarn lint if that's what you're saying. `yarn lint` definitely still lints the files in this package. What doesn't work is the `tsc` command from the root dir, hence the exclusion right now.